connect/runtime/src/main/java/org/apache/kafka/connect/storage/FileOffsetBackingStore.java

```diff
@@ -69,19 +69,19 @@ public synchronized void stop() {
 
     @SuppressWarnings("unchecked")
     private void load() {
-        try {
-            ObjectInputStream is = new ObjectInputStream(new FileInputStream(file));
+        try (ObjectInputStream is = new ObjectInputStream(new FileInputStream(file))) {
             Object obj = is.readObject();
-            if (!(obj instanceof HashMap))
+            if (!(obj instanceof HashMap)) {
+                is.close();
```

Comment:
Why is this needed? Doesn't the `try` ensure that the input stream will be closed?